core/src/main/scala/org/apache/spark/util/ClosureCleaner.scala

```diff
@@ -285,8 +285,6 @@ private[spark] object ClosureCleaner extends Logging {
         innerClasses.foreach { c => logDebug(s"     ${c.getName}") }
         logDebug(s" + outer classes: ${outerClasses.size}" )
         outerClasses.foreach { c => logDebug(s"     ${c.getName}") }
-        logDebug(s" + outer objects: ${outerObjects.size}")
-        outerObjects.foreach { o => logDebug(s"     $o") }
```

Comment:
You could keep this (and just change to `o.getClass()`).

Reply:
I removed this statement altogether because outerclasses are printed in the statement above.

Reply:
👍